__tests__/app.test.js

```diff
@@ -12,6 +12,14 @@ beforeEach(() => {
   return seed(data);
 });
 
+
+describe("/api/topics", () => {
+  test("404- test for bad URL", () => {
```

Comment:
Like that you have included this test, but it doesn't only work for /topics, so should probably have it's own separate desc block